src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceIntellisense/KustoIntellisenseHelper.cs

```diff
@@ -117,8 +117,11 @@ private static IReadOnlyList<Parameter> TranslateParameters(string parameters)
             var query = "let fn = " + parameters + " { };";
             var code = KustoCode.ParseAndAnalyze(query);
             var let = code.Syntax.GetFirstDescendant<LetStatement>();
-            var variable = let.Name.ReferencedSymbol as VariableSymbol;
-            var function = variable.Type as FunctionSymbol;
+            
+             FunctionSymbol function = let.Name.ReferencedSymbol is VariableSymbol variable
+                ? variable.Type as FunctionSymbol
+                : let.Name.ReferencedSymbol as FunctionSymbol;
```

Comment:
There were a few ReferencedSymbols that were failing to cast. This safely casts to FunctionSymbol